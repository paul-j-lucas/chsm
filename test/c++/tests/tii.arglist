-c | std::pair<int,int> p | 0
