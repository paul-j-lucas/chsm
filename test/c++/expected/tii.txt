decl="std::pair<int, int> $" name="p"
